### ISSUE TITLE
CIDER requires me to press up arrow twice to get into edit mode on my last message

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#10742]
Seemingly only after I switch rooms and send a message - subsequent attempts after an edit are fine, so long as I don't switch rooms

**Comment by jryans** [comment]
It appears the first press translates to focusing the composer, and then the next press is received by composer's event handling for edits. @bwindels, does that sound right?

**Comment by turt2live** [comment]
But I was already focused on the composer?

**Comment by jryans** [comment]
Ah hmm, I see...

**Comment by bwindels** [comment]
Hmm, I can't repro, but unsure if I understood correctly.

So the repro case is to switch room, and the first time you try to send a message, you need to hit arrow-up twice before it starts to edit the last event? Is this after having typed in the composer or when it's still empty?

Here I'm trying both and succeeding in editing with only one arrow-up (if cursor is at editor start after having typed):

![repro1](https://user-images.githubusercontent.com/274386/64351337-a55a5100-cfe9-11e9-9c65-9fe4d3794187.gif)

**Comment by turt2live** [comment]
The gif below is all I did. If you look carefully, there's a single frame where the cursor flashes - that's when I pressed the up arrow. The edit field coming up is the second keypress.

Steps were simply:
1. Switch rooms
2. Send message
3. Wait for it to send (just to be sure)
4. Press up arrow
5. Press up arrow again

![edit-new-composer mp4](https://user-images.githubusercontent.com/1190097/64353294-c2c3f680-cfba-11e9-8052-6ab76e8f4e75.gif)
